### PR TITLE
[4.0] featured articles update

### PR DESCRIPTION
**Pull request by brianteeman**
Fixes the tooltip size difference reported in #33876 and various codestyle issues

To test - code review AND check the tip sizes

This is a redo of #33894 due to merge conflicts

### CODE REVIEW

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on aaa8d35e352101d2ebb2bc6c4311b6e5b7c2d69d<br /><br />I can confirm that the tooltip on the stage button is properly placed with the same size text as the tooltips on the other buttons. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33927">issues.joomla.org/tracker/joomla-cms/33927</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on aaa8d35e352101d2ebb2bc6c4311b6e5b7c2d69d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33927">issues.joomla.org/tracker/joomla-cms/33927</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33927">issues.joomla.org/tracker/joomla-cms/33927</a>.</sub>

File: administrator/components/com_content/tmpl/featured/default.php [hunk 1]
```diff
@@ -40,6 +41,10 @@
 {
 	$orderingColumn = 'publish_down';
 }
+elseif (strpos($listOrder, 'modified') !== false)
+{
+	$orderingColumn = 'modified';
+}
```
Comment (richard67) [inline administrator/components/com_content/tmpl/featured/default.php:47]: @brianteeman The description of this PR doesn't mention this change. Just to be sure: Is it by purpose or by mistake?

Comment (richard67) [reply administrator/components/com_content/tmpl/featured/default.php:47]: In addition: Could you update your branch to 4.0-dev or push some other change pr let me know if I shall do that for you? Drone hangs and cannot be restarted manually.

Comment (brianteeman) [reply administrator/components/com_content/tmpl/featured/default.php:47]: it is on purpose. discovered when doing a diff between featured and non featured views

**Comment by brianteeman** [comment]
thank you

**Pull request merged**